redfish-core/lib/power_supply.hpp

```diff
@@ -62,6 +62,10 @@ inline void getPowerSupplyFirmwareVersion(
                     const std::variant<std::string> state) {
             if (ec)
             {
+                if (ec.value() == EBADR)
+                {
```

Comment:
Maybe add INFO or DEBUG journal log entry?

Reply:
@gtmills what you think

Reply:
Sure. 
<< "PowerSupply Firmware not found, skipping"